### PR TITLE
Fix pagination error

### LINKED ISSUE #385: Navigation bug in patient details screen
**Describe the bug**
Patient Navigation is failing.

**To Reproduce**
1. Run the app on Patient List / Try to navigate to next page

**Expected behavior**
Pagination should navigate the Patient List to the next list of Patients

**Screenshots**
N/A

**Smartphone (please complete the following information):**
Samsung Galaxy A2 emulator

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by rkodev**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #385

**Type**
Bug Fix

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by syedowaisali90 — APPROVED** [review]

File: android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt [hunk 1]
```diff
@@ -70,6 +70,8 @@ class SamplePatients {
     return patients
   }
 
+  fun getPatientItem(fhirPatient: Patient?) = fhirPatient?.let { createPatientItem(1, it) }
```
Comment (shoaibmushtaq25) [inline android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt:73]: is it ok to pass `id = 1` for every patient here? @Rkareko

Comment (syedowaisali90) [reply android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt:73]: It's not an `id` it's a `position` that is currently not in use

Comment (rkodev) [reply android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt:73]: @shoaibmushtaq25 This function has the same result as passing a list of 1 element to the function below. The id is a nonnull field 

```
 fun getPatientItems(fhirPatients: List<Patient>): List<PatientListViewModel.PatientItem> {
    val patients: MutableList<PatientListViewModel.PatientItem> = mutableListOf()

    // Create a list of PatientItems from fhirPatients. The display index is 1 based.
    fhirPatients
      .take(MAX_RESOURCE_COUNT)
      .mapIndexed { index, fhirPatient -> createPatientItem(index + 1, fhirPatient) }
      .let { patients.addAll(it) }

    // Return a cloned List
    return patients
  }
```

Ideally, we should remove the class `SamplePatients`

Comment (shoaibmushtaq25) [reply android/app/src/main/java/org/smartregister/fhircore/data/SamplePatients.kt:73]: got it, Thanks for the detailed answer. @Rkareko @syedowaisali90

**Pull request merged**